driverinference/main.py

```diff
@@ -131,7 +160,7 @@ def inference_colocated(self,
                             batch_size=[1],
                             device="GPU",
                             num_devices=1,
-                            net_ifname="lo",
+                            net_ifname="ipogif0",
```

Comment:
I think this name is probably very specific to Horizon

Reply:
Hmmm Ill consult Andrew for what to put there or look back at the original name